e2e-tests/commands/shared-commands/assertions/assert-heading-with-currency-name.js

```diff
@@ -5,20 +5,28 @@ import { heading } from '../../../pages/shared';
  * Assert a heading with a currency name and optional question mark
  * @param {String} pageTitle: Page title
  * @param {String} currencyName: Currency name
+ * @param {String} currencyName: Currency name
+ * @param {Function} selector: Cypress selector
  * @param {Boolean} withQuestionMark: Add a question mark to the expected text.
  */
 const assertHeadingWithCurrencyName = ({
-  pageTitle,
-  currencyName,
-  withQuestionMark = false,
+  pageTitle, currencyName, selector, withQuestionMark = false,
 }) => {
   let expected = `${pageTitle} ${currencyName}`;
 
   if (withQuestionMark) {
     expected += '?';
   }
 
-  cy.checkText(heading(), expected);
+  /**
+   * if selector provided, check text on selector
+   * else check heading text
+   */
+  if (selector) {
+    cy.checkText(selector, expected);
+  } else {
+    cy.checkText(heading(), expected);
+  }
```

Comment:
Not sure if it should be done this way if it's testing for a legend?

Because now the command is called `assertHeadingWithCurrencyName` but it won't always assert a heading